dea_waterbodies/make_time_series.py

```diff
@@ -165,28 +165,63 @@ def get_shapes(config_dict: dict,
 
 @click.command()
 @click.argument('ids', required=False, default='')
-@click.option('--config', '-c', type=click.Path(), default=None)
-@click.option('--shapefile', type=click.Path(), default=None)
-@click.option('--start', type=str, default=None)
-@click.option('--end', type=str, default=None)
-@click.option('--size', type=click.Choice(['ALL', 'SMALL', 'HUGE']),
-              default='ALL')
-@click.option('--missing-only/--not-missing-only', default=False)
-@click.option('--skip', type=click.Path(), default=None)
+@click.option('--config', '-c', type=click.Path(), default=None,
+              help='Path to a config.ini file. A config file can be provided '
+              'instead of using the command line options provided.')
+@click.option('--shapefile', type=click.Path(), default=None,
+              help='REQUIRED. Path to the waterbody polygon shapefile (or '
+              'geojson) you want to run the time series generation for.')
+@click.option('--output', type=click.Path(), default=None,
+              help='REQUIRED. Path to the output directory for the waterbody '
+              'timeseries.')
 @click.option('--time-span', type=click.Choice(['ALL', 'APPEND', 'CUSTOM']),
-              default='ALL')
-@click.option('--output', type=click.Path(), default=None)
+              default='ALL',
+              help='--time-span sets the time range for the waterbody '
+              'timeseries queries. --time-span will default to ALL if no '
+              'other option is specified. If you select APPEND, then only '
+              'times since the latest dates in the waterbody timeseries will '
+              'be run. If --time-span = CUSTOM, then --start and --start must '
+              'also be specified.')
+@click.option('--start', type=str, default=None,
+              help='Date string. E.g. 2019-01-01. '
+              'The start date for the waterbody timeseries query. If --start '
+              'is provided --end must also be provided.')
+@click.option('--end', type=str, default=None,
+              help='Date string. E.g. 2019-12-01. '
+              'The end date for the waterbody timeseries query. If --end is '
+              'provided --start must also be provided.')
+@click.option('--missing-only/--not-missing-only', default=False,
+              help='This option specifies whether you want to only run '
```

Comment:
-> "Whether to only run waterbody polygons that...". Don't need to specify that it's an option.

Reply:
Changed.